### ISSUE TITLE
Unexpected token in /core/tasks/browser-sync.js

### ISSUE THREAD

**Issue by driesderoeck** [mono-company/bedrock#89]
Error received:
```/bedrock/core/tasks/browser-sync.js:18
    proxy: `localhost:${global.expressPort}`
           ^
SyntaxError: Unexpected token ILLEGAL```
Solved by using ' instead of `

**Comment by Wolfr** [comment]
I think your babel compiler is not working properly which is causing the same error as #88.

**Comment by driesderoeck** [comment]
Resolved after node update to v5.4.1
